.github/workflows/build.yaml

```diff
@@ -0,0 +1,23 @@
+name: Build and Test
+
+on: [push, pull_request]
```

Comment:
usually you don't want full `push` or it will double-run any pull requests -- this is usually what I use:

```yaml
on:
 push:
   branches: [main, test-me-*]
   tags:
 pull_request:
```